src/ameba/rule/lint/bad_directive.cr

```diff
@@ -0,0 +1,54 @@
+module Ameba::Rule::Lint
+  # A rule that reports incorrect comment directives for Ameba.
+  #
+  # For example, the user can mistakenly add a directive
+  # to disable a rule that even doesn't exist:
+  #
+  # ```
+  # # ameba:disable BadRuleName
+  # def foo
+  #   :bar
+  # end
+  # ```
+  #
+  # YAML configuration example:
+  #
+  # ```
+  # Lint/BadDirective:
+  #   Enabled: true
+  # ```
+  #
+  struct BadDirective < Base
+    properties do
+      description "Reports bad comment directives"
+    end
+
+    AVAILABLE_ACTIONS = InlineComments::Action.names.join(',', &.downcase)
+    ALL_RULE_NAMES    = Rule.rules.map(&.rule_name)
```

Comment:
If someone extends or adds custom rules will this catch those?

Reply:
Good question. Yes, this method returns all available rules at compile time.